platform/nebulas/api.go

```diff
@@ -56,3 +77,41 @@ func NormalizeTx(srcTx Transaction) blockatlas.Tx {
 		},
 	}
 }
+
+func NormalizeNasTx(srcTx NasTransaction, block NasBlock) blockatlas.Tx {
```

Comment:
This is the simular to https://github.com/trustwallet/blockatlas/blob/30f6ace4f32d7d5212b2ea9ad13c4a268aaa6802/platform/nebulas/api.go#L58

Reply:
An existing GetTxsByAddress() api in this file uses the function NormalizeTx() and it seems to be calling a different api endpoint not included in the documentation you sent us. here https://github.com/nebulasio/wiki/blob/master/rpc.md

So since I don't have the documentation of the api that GetTxsByAddress() api is calling, I didn't want to break the code that's why I created another that normalize specifically for my api.

But I will remove the new one i created and just update the old one that's there.

Reply:
Done

Reply:
I will include a test for this new normalizer function using its own sample inputs and outputs and you will see that the format is different from the first one.

Reply:
Added test for new normalizer function